src/deprecation.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_DEPRECATION_H
+#define ZCASH_DEPRECATION_H
+
+// Deprecation policy is 4th third-Tuesday after a release
+static const int APPROX_RELEASE_HEIGHT = 115000;
+static const int WEEKS_UNTIL_DEPRECATION = 18;
```

Comment:
Add a comment like: `// We use 18 weeks to ensure auto-deprecation is *always* > 4 months, barring block time skew issues.`